posthog/request.py

```diff
@@ -63,7 +63,7 @@ def _process_response(
 
 def decide(api_key: str, host: Optional[str] = None, gzip: bool = False, timeout: int = 15, **kwargs) -> Any:
     """Post the `kwargs to the decide API endpoint"""
-    res = post(api_key, host, "/decide/?v=2", gzip, timeout, **kwargs)
+    res = post(api_key, host, "/decide/?v=3", gzip, timeout, **kwargs)
```

Comment:
is the resiliency stuff in a separate pr? 😅 (I'm hoping you're handling that too 😬 )

Reply:
not anymore! 😈

Reply:
Sounds good to me!